ArduSub/ArduSub.cpp

```diff
@@ -46,6 +46,31 @@ SCHED_TASK_CLASS arguments:
  */
 
 const AP_Scheduler::Task Sub::scheduler_tasks[] = {
+    // update INS immediately to get current gyro data populated
+    FAST_TASK_CLASS(AP_InertialSensor, &sub.ins, update),
```

Comment:
better to just call fast_loop fn so simpler change for sub and blimp

Reply:
Yep